When you rebase (so that the test will run), you will get the error :+1: 

INFO] Running org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging.JNDIBindingsNoAppNameTestCase
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.016 s <<< FAILURE! - in org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging.JNDIBindingsNoAppNameTestCase
[ERROR] org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging.JNDIBindingsNoAppNameTestCase  Time elapsed: 0.015 s  <<< ERROR!
java.lang.IllegalArgumentException: org.jboss.arquillian.protocol.servlet.v_3.ServletProtocolDeploymentPackager can not handle archive of type org.jboss.shrinkwrap.impl.base.spec.JavaArchiveImpl
	at org.jboss.arquillian.protocol.servlet.v_3.ServletProtocolDeploymentPackager.generateDeployment(ServletProtocolDeploymentPackager.java:79)
	at org.jboss.arquillian.container.test.impl.client.deployment.DeploymentGenerator.buildTestableDeployments(DeploymentGenerator.java:193)
	at org.jboss.arquillian.container.test.impl.client.deployment.DeploymentGenerator.createTestableDeployments(DeploymentGenerator.java:148)
	at org.jboss.arquillian.container.test.impl.client.deployment.DeploymentGenerator.generateDeployment(DeploymentGenerator.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:96)
	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:85)
	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:143)
	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:114)
	at org.jboss.arquillian.core.impl.EventImpl.fire(EventImpl.java:67)
	at org.jboss.arquillian.container.test.impl.client.ContainerEventController.execute(ContainerEventController.java:100)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:96)
	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:85)
	at org.jboss.arquillian.test.impl.TestContextHandler.createClassContext(TestContextHandler.java:92)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:96)
	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:92)
	at org.jboss.arquillian.test.impl.TestContextHandler.createSuiteContext(TestContextHandler.java:73)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:96)
	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:92)
	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:143)
	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:114)
	at org.jboss.arquillian.test.impl.EventTestRunnerAdaptor.beforeClass(EventTestRunnerAdaptor.java:87)
	at org.jboss.arquillian.junit.Arquillian$2.evaluate(Arquillian.java:202)
	at org.jboss.arquillian.junit.Arquillian.multiExecute(Arquillian.java:431)
	at org.jboss.arquillian.junit.Arquillian.access$200(Arquillian.java:55)
	at org.jboss.arquillian.junit.Arquillian$3.evaluate(Arquillian.java:219)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.jboss.arquillian.junit.Arquillian.run(Arquillian.java:167)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)

--- a/modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi/logging/Hello.java
+++ b/modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi/logging/Hello.java
@@ -1,0 +1,7 @@
+package org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging;
+
+import org.jboss.eap.additional.testsuite.annotations.EapAdditionalTestsuite;
+
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/ejb/src/main/java#16.0.0.Beta1", "modules/testcases/jdkAll/Eap72x/ejb/src/main/java#7.2.2", "modules/testcases/jdkAll/Eap72x-Proposed/ejb/main/java#7.2.2"})
+public interface Hello {
+}

--- a/modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi/logging/HelloBean.java
+++ b/modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi/logging/HelloBean.java
@@ -1,0 +1,12 @@
+package org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging;
+
+import org.jboss.eap.additional.testsuite.annotations.EapAdditionalTestsuite;
+
+import javax.ejb.Remote;
+import javax.ejb.Stateless;
+
+@Stateless(name="Hello")
+@Remote(Hello.class)
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/ejb/src/main/java#16.0.0.Beta1", "modules/testcases/jdkAll/Eap72x/ejb/src/main/java#7.2.2", "modules/testcases/jdkAll/Eap72x-Proposed/ejb/main/java#7.2.2"})
+public class HelloBean implements Hello {
+}

--- a/modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi/logging/JNDIBindingsNoAppNameTestCase.java
+++ b/modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi/logging/JNDIBindingsNoAppNameTestCase.java
@@ -1,0 +1,115 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2019, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.jboss.eap.additional.testsuite.annotations.EapAdditionalTestsuite;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.MalformedInputException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Automated test for [ JBEAP-16550 ] - Tests if JNDI bindings is correctly built in case there is no appName.
+ *
+ * @author Daniel Cihak
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/ejb/src/main/java#16.0.0.Beta1", "modules/testcases/jdkAll/Eap72x/ejb/src/main/java#7.2.2", "modules/testcases/jdkAll/Eap72x-Proposed/ejb/main/java#7.2.2"})
+public class JNDIBindingsNoAppNameTestCase {
+
+    private static final String JAR_NAME = "ejb-jndi";
+    private static String HOST = TestSuiteEnvironment.getServerAddress();
+    private static int PORT = TestSuiteEnvironment.getHttpPort();
+
+    @Deployment
+    public static Archive<?> createJar() {
+        JavaArchive jar = ShrinkWrap.create(JavaArchive.class, JAR_NAME + ".jar");
+        jar.addClasses(JNDIBindingsNoAppNameTestCase.class, Hello.class, HelloBean.class);
+        return jar;
+    }
+
+    @Test
+    public void testJNDIBindingsNoAppName() throws Exception {
+        boolean passed = false;
+        Context ctx = getInitialContext(HOST, PORT);
+        Hello ejb = (Hello) ctx.lookup("ejb:/ejb-jndi/Hello!org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging.Hello");
+        Assert.assertNotNull("Null object returned for local business interface lookup in the ejb namespace", ejb);
+        List<String> lines = this.readServerLogLines();
+        int i = 0;
+        while (i < lines.size() && !passed) {
+            String line = lines.get(i);
+            if (line.contains("ejb:/test/Hello!org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging.Hello")) {
+                passed = true;
+            }
+            i++;
+        }
+        Assert.assertTrue(passed);
+    }
+
+    private static Context getInitialContext(String host, Integer port)  throws NamingException {
+        Properties props = new Properties();
+        props.put(Context.INITIAL_CONTEXT_FACTORY, "org.wildfly.naming.client.WildFlyInitialContextFactory");
+        props.put(Context.PROVIDER_URL, String.format("%s://%s:%d", "remote+http", host, port));
+        return new InitialContext(props);
+    }
+
+    private static List<String> readServerLogLines() {
+        String jbossHome = System.getProperty("jboss.home");
+        String logPath = String.format("%s%sstandalone%slog%sserver.log", jbossHome,
+                (jbossHome.endsWith(File.separator) || jbossHome.endsWith("/")) ? "" : File.separator,
+                File.separator, File.separator);
+        logPath = logPath.replace('/', File.separatorChar);
+        try {
+            return Files.readAllLines(Paths.get(logPath));
+        } catch (MalformedInputException e1) {
+            try {
+                return Files.readAllLines(Paths.get(logPath), StandardCharsets.ISO_8859_1);
+            } catch (IOException e4) {
+                throw new RuntimeException("Server logs has not standard Charsets (UTF8 or ISO_8859_1)");
+            }
+        } catch (IOException e) {
+            // server.log file is not created, it is the same as server.log is empty
+        }
+        return new ArrayList<>();
+    }
+}

--- a/modules/src2/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi.logging/Hello.java
+++ b/modules/src2/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi.logging/Hello.java
@@ -1,0 +1,7 @@
+package org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging;
+
+import org.jboss.eap.additional.testsuite.annotations.EapAdditionalTestsuite;
+
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/ejb/src/main/java#16.0.0.Beta1", "modules/testcases/jdkAll/Eap72x/ejb/src/main/java#7.2.2", "modules/testcases/jdkAll/Eap72x-Proposed/ejb/main/java#7.2.2"})
+public interface Hello {
+}

--- a/modules/src2/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi.logging/HelloBean.java
+++ b/modules/src2/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi.logging/HelloBean.java
@@ -1,0 +1,12 @@
+package org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging;
+
+import org.jboss.eap.additional.testsuite.annotations.EapAdditionalTestsuite;
+
+import javax.ejb.Remote;
+import javax.ejb.Stateless;
+
+@Stateless(name="Hello")
+@Remote(Hello.class)
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/ejb/src/main/java#16.0.0.Beta1", "modules/testcases/jdkAll/Eap72x/ejb/src/main/java#7.2.2", "modules/testcases/jdkAll/Eap72x-Proposed/ejb/main/java#7.2.2"})
+public class HelloBean implements Hello {
+}

--- a/modules/src2/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi.logging/JNDIBindingsNoAppNameTestCase.java
+++ b/modules/src2/main/java/org/jboss/additional/testsuite/jdkall/present/ejb/jndi.logging/JNDIBindingsNoAppNameTestCase.java
@@ -1,0 +1,115 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2019, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.jboss.eap.additional.testsuite.annotations.EapAdditionalTestsuite;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.MalformedInputException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Automated test for [ JBEAP-16550 ] - Tests if JNDI bindings is correctly built in case there is no appName.
+ *
+ * @author Daniel Cihak
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/ejb/src/main/java#16.0.0.Beta1", "modules/testcases/jdkAll/Eap72x/ejb/src/main/java#7.2.2", "modules/testcases/jdkAll/Eap72x-Proposed/ejb/main/java#7.2.2"})
+public class JNDIBindingsNoAppNameTestCase {
+
+    private static final String JAR_NAME = "ejb-jndi";
+    private static String HOST = TestSuiteEnvironment.getServerAddress();
+    private static int PORT = TestSuiteEnvironment.getHttpPort();
+
+    @Deployment
+    public static Archive<?> createJar() {
+        JavaArchive jar = ShrinkWrap.create(JavaArchive.class, JAR_NAME + ".jar");
+        jar.addClasses(JNDIBindingsNoAppNameTestCase.class, Hello.class, HelloBean.class);
+        return jar;
+    }
+
+    @Test
+    public void testJNDIBindingsNoAppName() throws Exception {
+        boolean passed = false;
+        Context ctx = getInitialContext(HOST, PORT);
+        Hello ejb = (Hello) ctx.lookup("ejb:/ejb-jndi/Hello!org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging.Hello");
+        Assert.assertNotNull("Null object returned for local business interface lookup in the ejb namespace", ejb);
+        List<String> lines = this.readServerLogLines();
+        int i = 0;
+        while (i < lines.size() && !passed) {
+            String line = lines.get(i);
+            if (line.contains("ejb:/test/Hello!org.jboss.additional.testsuite.jdkall.present.ejb.jndi.logging.Hello")) {
+                passed = true;
+            }
+            i++;
+        }
+        Assert.assertTrue(passed);
+    }
+
+    private static Context getInitialContext(String host, Integer port)  throws NamingException {
+        Properties props = new Properties();
+        props.put(Context.INITIAL_CONTEXT_FACTORY, "org.wildfly.naming.client.WildFlyInitialContextFactory");
+        props.put(Context.PROVIDER_URL, String.format("%s://%s:%d", "remote+http", host, port));
+        return new InitialContext(props);
+    }
+
+    private static List<String> readServerLogLines() {
+        String jbossHome = System.getProperty("jboss.home");
+        String logPath = String.format("%s%sstandalone%slog%sserver.log", jbossHome,
+                (jbossHome.endsWith(File.separator) || jbossHome.endsWith("/")) ? "" : File.separator,
+                File.separator, File.separator);
+        logPath = logPath.replace('/', File.separatorChar);
+        try {
+            return Files.readAllLines(Paths.get(logPath));
+        } catch (MalformedInputException e1) {
+            try {
+                return Files.readAllLines(Paths.get(logPath), StandardCharsets.ISO_8859_1);
+            } catch (IOException e4) {
+                throw new RuntimeException("Server logs has not standard Charsets (UTF8 or ISO_8859_1)");
+            }
+        } catch (IOException e) {
+            // server.log file is not created, it is the same as server.log is empty
+        }
+        return new ArrayList<>();
+    }
+}

